README.md

```diff
@@ -1 +1,166 @@
+
+- Write JavaScript code using functions
+- Practice TDD with JavaScript and the Jest testing framework
+
+## Getting Started
+After forking and cloning this repo you should `cd` to the project directory and run:
+
+```bash
+$ npm install
+```
+
+Similar to `bundle install` in Ruby projects, this makes the npm package manager download and install any dependencies for the project (such as Jest).
+
+### Tests
+We have provided unit tests for you to run. A complete project will pass all provided tests.
+
+To run the tests, in the command line, navigate to the project root and then run:
+
+```bash
+$ npm test
+```
+
+After your tests have run there is a short table outputting the coverage summary for each file. You can view complete coverage details by running:
+
+```bash
+$ npm run coverage
+```
+
+This is shorthand for the command `open coverage/lcov-report/index.html` and will open the coverage report in your default web browser.
+
+### Driver Code
+We have provided some driver code for your Adagrams game in the files `wave-1-game.rb`, `wave-2-game.rb`, `wave-3-game.rb`, and `wave-4-game.rb`. Running `$ ruby wave-1-game.rb` will begin a command-line game that uses your Adagrams code. The boilerplate code will break the first time you run it: working through the waves specified below should create a running version of the game. **Implementing code to make this game run is not a substitute for making the tests pass**. It is simply there for you and your pair to reference how the Game may run during each wave, to have better perspective of what your program can do, and to get some practice reading other peoples' code. We fully expect you to create the specified methods to specification and making the tests pass.
+
+### Project Structure
+This repository has a baseline structure for the project which includes several files. You will only need to modify one of them:
+
+File | Description
+--- | ---
+src/adagrams.js | This is the project code, you implementation should be written here as functions within the `Adagrams` object.
+specs/adagrams.spec.js | This file contains the unit tests for all functions you must implement.
+package.json | This is the [npm project description file](https://docs.npmjs.com/getting-started/using-a-package.json) for this project. It includes all of the dependencies, much like `Gemfile` in our Ruby projects.
+package-lock.json | This is equivalent to the `Gemfile.lock` file in our Ruby projects.
+jest.config.js | This is a configuration file for the Jest testing framework.
+babel.config.js | This is a configuration file for the [Babel](https://babeljs.io/) compiler. For a short summary of why we use it, [click here](https://github.com/AdaGold/backbone-baseline#major-components)
+
+## Implementation Requirements
+### Wave 1
+Our first job is to build a hand of 10 letters for the user. To do so, add a method called `Adagrams.drawLetters` in `src/adagrams.js`. This method should have the following properties:
```

Comment:
Hm, I have not-strong thoughts about taking out the phrase "for the user"... since this won't be a "user-facing" piece of functionality, could we take the phrase out?